schema/generate/30-schema_update_32.sql

```diff
@@ -0,0 +1,34 @@
+CREATE TABLE "augur_data"."pull_request_analysis" (
+  "pull_request_analysis_id" serial8,
+  "pull_request_id" int8,
+  "merge_probability" float8,
+  "mechanism" varchar,
+  "tool_source" varchar,
+  "tool_version" varchar,
+  "data_source" varchar,
+  "data_collection_date" timestamptz NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  PRIMARY KEY ("pull_request_analysis_id")
+)
+;
+
+ALTER TABLE "augur_data"."pull_request_analysis" OWNER TO "augur";
+
+COMMENT ON COLUMN "augur_data"."pull_request_analysis"."pull_request_id" IS 'It would be better if the pull request worker is run first to fetch the latest PRs before analyzing';
+
+COMMENT ON COLUMN "augur_data"."pull_request_analysis"."merge_probability" IS 'Indicates the probability of the PR being merged';
+
+COMMENT ON COLUMN "augur_data"."pull_request_analysis"."mechanism" IS 'the ML model used for prediction (It is XGBoost Classifier at present)';
+
+ALTER TABLE "augur_data"."discourse_insights" ADD CONSTRAINT "fk_discourse_insights_message_1" FOREIGN KEY ("msg_id") REFERENCES "augur_data"."message" ("msg_id");
```

Comment:
When trying to build the schema on Docker, this script throws the following error:
```
2020-11-28 16:37:45.713 UTC [334] ERROR:  constraint "fk_discourse_insights_message_1" for relation "discourse_insights" already exists
2020-11-28 16:37:45.713 UTC [334] STATEMENT:  ALTER TABLE "augur_data"."discourse_insights" ADD CONSTRAINT "fk_discourse_insights_message_1" FOREIGN KEY ("msg_id") REFERENCES "augur_data"."message" ("msg_id");
psql:/docker-entrypoint-initdb.d/30-schema_update_32.sql:22: ERROR:  constraint "fk_discourse_insights_message_1" for relation "discourse_insights" already exists
make: *** [docker-run-database] Error 3
```
This fatally terminates the build and makes the container unusable. Please fix this.